test/python/circuit/test_circuit_load_from_qpy.py

```diff
@@ -1043,9 +1054,17 @@ def test_open_controlled_gate(self):
 
     def test_standard_control_gates(self):
         """Test standard library controlled gates."""
-        qc = QuantumCircuit(3)
+        qc = QuantumCircuit(6)
         mcu1_gate = MCU1Gate(np.pi, 2)
+        mcx_gate = MCXGate(5)
+        mcx_gray_gate = MCXGrayCode(5)
+        mcx_recursive_gate = MCXRecursive(5)
+        mcx_vchain_gate = MCXVChain(5)
         qc.append(mcu1_gate, [0, 2, 1])
+        qc.append(mcx_gate, list(range(0, 6)))
+        qc.append(mcx_gray_gate, list(range(0, 6)))
+        qc.append(mcx_recursive_gate, list(range(0, 6)))
```

Comment:
It looks like `MCXRecursive` also has an ancilla qubit in addition to n control qubits.
```suggestion
        qc.append(mcx_recursive_gate, list(range(7)))
```